package.json

```diff
@@ -29,6 +29,7 @@
   "devDependencies": {
     "@antora/cli": "^2.1.2",
     "@antora/site-generator-default": "^2.1.2",
+    "@typechain/ethers-v5": "^6.0.5",
```

Comment:
bit of a pain that `typechain` needs ethers.

Reply:
This isn't ethers, it's just the ethers typechain plugin, so it should have a pretty small footprint (I hope!)